src/Sarif.Multitool.Library/Rules/RuleResources.Designer.cs

```diff
@@ -78,6 +78,30 @@ internal class RuleResources {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to The &apos;$schema&apos; property must refer to the final version of the SARIF 2.1.0 schema. This enables IDEs to provide Intellisense for SARIF log files.
+        ///
+        ///The SARIF standard was developed over several years, and many intermediate versions of the schema were produced. Now that the standard is final, only the OASIS standard version of the schema is valid..
+        /// </summary>
```

Comment:
Remove extra period at end of sentence.

Reply:
Second one is added by VS, not part of the string.